help_to_heat/frontdoor/views.py

```diff
@@ -371,6 +375,14 @@ def get(self, request, session_id, page_name, unsaved_data=None, errors=None, is
         if data_with_get_answers != data:
             save_answer(session_id, page_name, data_with_get_answers)
 
+        if page_name not in schemas.routing_overrides:
+            try:
+                # ensure the user has answers to complete the journey to this page
+                calculate_journey(answers, to_page=page_name)
+            except CouldNotCalculateJourneyException as e:
+                logger.exception(e)
```

Comment:
Ok, so to confirm, this is the equivalent to what we currently do, we just send them to a different page?

Reply:
yes, to be specific:
- currently, if you try to view a page that's not on your route, you will see an error page
- this happens when trying to calculate the previous page
- calculating the previous page requires finding a route to your page, and going 1 back
- the code will fail on trying to find a route to your page
- this has been moved to an explicit check
- so functionality should be identical to before

Reply:
Yes, and in particular we'll still get an error logged for it?

Reply:
yes (the one in this diff)